internal/app/machined/pkg/controllers/k8s/control_plane_static_pod.go

```diff
@@ -199,7 +200,6 @@ func (ctrl *ControlPlaneStaticPodController) manageAPIServer(ctx context.Context
 		fmt.Sprintf("--etcd-certfile=%s", filepath.Join(constants.KubernetesAPIServerSecretsDir, "etcd-client.crt")),
 		fmt.Sprintf("--etcd-keyfile=%s", filepath.Join(constants.KubernetesAPIServerSecretsDir, "etcd-client.key")),
 		fmt.Sprintf("--etcd-servers=%s", strings.Join(cfg.EtcdServers, ",")),
-		"--insecure-port=0",
```

Comment:
I don't remember what was the default value in kubernetes 1.20-1.21?

Reply:
It was ignored in 1.20: https://github.com/kubernetes/kubernetes/blob/af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38/cmd/kube-apiserver/app/options/options.go#L159-L163